interface/forms/aftercare_plan/new.php

```diff
@@ -26,153 +28,197 @@
-    </tr>
-    <tr>
-        <td align="right" class="forms">1.</td>
-        <td colspan="3"><textarea name="goal_a_acute_intoxication" rows="2" cols="80" wrap="virtual name"><?php echo text($obj["goal_a_acute_intoxication"]);?></textarea></td>
-
-    </tr>
-    <tr>
-        <td align="right" class="forms">2.</td>
-        <td colspan="3"><textarea name="goal_a_acute_intoxication_I" rows="2" cols="80" wrap="virtual name"><?php echo text($obj["goal_a_acute_intoxication_I"]);?></textarea></td>
-
-    </tr>
-    <tr>
-        <td align="right" class="forms">3.</td>
-        <td colspan="3"><textarea name="goal_a_acute_intoxication_II" rows="2" cols="80" wrap="virtual name"><?php echo text($obj["goal_a_acute_intoxication_II"]);?></textarea></td>
-
-
-    <tr>
-
-        <td class="forms-subtitle" colspan="4"><B><?php echo xlt('Goal B');?>:</B>&nbsp;<?php  echo xlt('Emotional / Behavioral Conditions & Complications'); ?></td>
-
-    </tr>
-    <tr>
-        <td align="right" class="forms">1.</td>
-        <td colspan="3"><textarea name="goal_b_emotional_behavioral_conditions" rows="2" cols="80" wrap="virtual name"><?php echo text($obj["goal_b_emotional_behavioral_conditions"]);?></textarea></td>
-
-    </tr>
-    <tr>
-        <td align="right" class="forms">2.</td>
-        <td colspan="3"><textarea name="goal_b_emotional_behavioral_conditions_I" rows="2" cols="80" wrap="virtual name"><?php echo text($obj["goal_b_emotional_behavioral_conditions_I"]);?></textarea></td>
-
-    </tr>
-
-
-        <td class="forms-subtitle" colspan="4"><B><?php echo xlt('Goal C');?>:</B>&nbsp;<?php  echo xlt('Relapse Potential'); ?></td>
-
-    </tr>
-    <tr>
-        <td align="right" class="forms">1.</td>
-        <td colspan="3"><textarea name="goal_c_relapse_potential" rows="2" cols="80" wrap="virtual name"><?php echo text($obj["goal_c_relapse_potential"]);?></textarea></td>
-
-    </tr>
-    <tr>
-        <td align="right" class="forms">2.</td>
-        <td colspan="3"><textarea name="goal_c_relapse_potential_I" rows="2" cols="80" wrap="virtual name"><?php echo text($obj["goal_c_relapse_potential_I"]);?></textarea></td>
-
-    </tr>
-
-    <tr>
-        <td align="left colspan="3" style="padding-bottom:7px;"></td>
-    </tr>
-    <tr>
-        <td></td>
-    <td><input type='submit' value='<?php echo xla('Save'); ?>' class='button-css' />&nbsp;
-    <input type='button' value='<?php echo xla('Print'); ?>' id='printbutton' class='button-css' />&nbsp;
-    <input type='button' class='button-css' value='<?php echo xla('Cancel'); ?>'
- onclick="parent.closeTab(window.name, false)" /></td>
-    </tr>
-</table>
-</form>
+<div class="container">
+    <div class="row">
+
+                <h1 class="forms-title"><?php echo xlt('After Care Planning'); ?></h1>
+
+        <div class="col-md-12 mt-4">
+            <?php
+            echo "<form method='post' name='my_form' " .
+                "action='$rootdir/forms/aftercare_plan/save.php?id=" . attr_url($formid) . "'>\n";
```

Comment:
Since you're refactoring the form, this is a good opportunity to reduce the amount of HTML being echo'd. This is better as:

```
<?php 
$form_action = "$rootdir/forms/aftercare_plan/save.id=" . attr_url($formId);
</php
<form method="post" name="my_form" action="<?php echo $form_action;?>">
```
not a huge deal, but the less HTML we have embedded in PHP, the better.